demophin.py

```diff
@@ -53,7 +66,7 @@ def bare_grmkey(grmkey):
 @view('main')
 def main(grmkey):
     grm = get_grammar(grmkey)
-    sent = request.query.get('sentence')
+    sent = request.query.getunicode('sentence')
```

Comment:
What is the purpose here? Are you running with Python 2? I don't see a problem with always ensuring the sentence is unicode, but the implementation of `getunicode()` worries me a bit (first encode as `latin1`, then decode again? http://bottlepy.org/docs/dev/_modules/bottle.html#FormsDict.getunicode)